cfpb/cfgov-refresh
# Ask CFPB admin: update edit button layout

Small updates to edit button layout

## Testing

1. Check out branch
2. Verify edit buttons on answer pages

## Screenshots
<img width="928" alt="screen shot 2017-07-12 at 4 02 28 pm" src="https://user-images.githubusercontent.com/778171/28143687-d14038cc-671b-11e7-8bb6-ee68fd36ae51.png">
<img width="1045" alt="screen shot 2017-07-12 at 4 02 17 pm" src="https://user-images.githubusercontent.com/778171/28143688-d142fb70-671b-11e7-8e85-a3c7ad007a31.png">


## Checklist
